### DISCUSSION TITLE
Hello, I want to know how I can be sure that the loop diagram is obtained by using this algorithm. Then how can I be sure that the robot has returned to its original position? If I only get the loop diagram, how can I compare it with the scan context? After all, scan context It is possible to obtain the specific pose of the robot when it returns to its original position.

### DISCUSSION THREAD

**Discussion by LL884800** [PRBonn/MapClosures#34] [General]
Hello, I want to know how I can be sure that the loop diagram is obtained by using this algorithm. Then how can I be sure that the robot has returned to its original position? If I only get the loop diagram, how can I compare it with the scan context? After all, scan context It is possible to obtain the specific pose of the robot when it returns to its original position.

**Comment by saurabh1002** [comment]
Being adressed in #33
